code/modules/multiz/movement.dm

```diff
@@ -462,6 +462,11 @@
 		"With a loud thud, you land on \the [loc]!", "You hear a thud!")
 
 	var/z_velocity = 5*(levels_fallen**2)
+
+	//Since maint drones don't have a fall_mod that can be modified, they will have a check here
+	if(isDrone(src))
+		damage_mod = 0.25
```

Comment:
That should be implemented in a subtype of /mob/living

Reply:
like a different version of this method? From what I saw from code, damage_mod is always given in this method, and set as 1